node/globals.ts

```diff
@@ -67,6 +67,9 @@ declare global {
     isGift: boolean
     parentItemIndex: number
     parentAssemblyBinding: string
+    productCategoryIds: string
+    priceTags: []
```

Comment:
same in here, maybe  ?
```suggestion
    priceTags: string[]
```

Reply:
Price tags is an array of objects, how could I define this?